From 75b59483f5122366ec4c31e54df1c0647cac096c Mon Sep 17 00:00:00 2001
From: Tsz-Wo Nicholas Sze <szetszwo@hortonworks.com>
Date: Tue, 24 Nov 2015 14:34:51 -0800
Subject: [PATCH 1111/2748] HDFS-9434. Recommission a datanode with 500k
 blocks may pause NN for 30 seconds for printing
 info log messags.

(cherry picked from commit d2518fd5627220736e691b4a55638fc1f57d612e)

Change-Id: I41cd48ddbc5081694b7402789d992ceb76ee6d2e
---
 .../hdfs/server/blockmanagement/BlockManager.java  |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java
index a27ebd5..1f1505c 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/BlockManager.java
@@ -2929,11 +2929,11 @@ private void processOverReplicatedBlock(final Block block,
     for(DatanodeStorageInfo storage : blocksMap.getStorages(block, State.NORMAL)) {
       final DatanodeDescriptor cur = storage.getDatanodeDescriptor();
       if (storage.areBlockContentsStale()) {
-        LOG.info("BLOCK* processOverReplicatedBlock: " +
-            "Postponing processing of over-replicated " +
-            block + " since storage + " + storage
-            + "datanode " + cur + " does not yet have up-to-date " +
-            "block information.");
+        if (LOG.isTraceEnabled()) {
+          LOG.trace("BLOCK* processOverReplicatedBlock: Postponing " + block
+              + " since storage " + storage
+              + " does not yet have up-to-date information.");
+        }
         postponeBlock(block);
         return;
       }
-- 
1.7.9.5

